e2e/yarn.lock

```diff
@@ -0,0 +1,1354 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We should add `e2e` in workspaces

Reply:
done ,will push soon, working on adding more envs :)